### PR TITLE
Update to wallet recovery in the Savings Guide

### PR DESCRIPTION
**Pull request by steynviljoen67**
For this pull request, I did the following based on [this task](https://github.com/BitcoinDesign/Guide/issues/991):

- Combined the [Signing device key replacement](https://bitcoin.design/guide/savings-wallet/#signing-device-key-replacement) and [Cloud storage key recovery & replacement](https://bitcoin.design/guide/savings-wallet/#cloud-storage-key-recovery--replacement) sections into one
- Gave it a common heading, "Wallet Recovery and Key Replacement"
- Separated the different scenarios under which users may lose access to their wallet or keys, into sub headings
- Outlined the methods to recover/replace a key for each scenario

### CODE REVIEW

**Comment by steynviljoen67** [comment]
Note that we don't have prototypes for all scenarios listed in the PR. We'll have to elaborate on them at some point.

File: guide/savings-wallet.md [hunk 1]
```diff
@@ -221,24 +221,53 @@ For larger transactions, the wallet application provider will no longer automati
 
 {% include image-gallery.html pages = page.images_send_manual %}
 
-### Signing device key replacement
+### Wallet Recovery and Key Replacement
 
-Earlier we mentioned that one of the keys is saved on the user's mobile device and backed up to the cloud. To prevent unauthorized access to this backup, a PIN is created during wallet set-up and used to encrypt and decrypt the backup.
+Restoring a bitcoin wallet or replacing keys may be necessary for various reasons. The user may have lost their mobile or signing device, or they might have lost access to their backup cloud account. Regardless of the reason, wallet applications must offer a straightforward recovery option for users to regain access to their funds.
```
Comment (danielnordh) [inline guide/savings-wallet.md:226]:
```suggestion
Restoring a bitcoin wallet or replacing keys may be necessary for various reasons. The user may have lost their mobile phone, signing device, or access to their cloud account backup. Regardless of the reason, wallet applications must offer a straightforward recovery option for users to regain access to their funds.
```

File: guide/savings-wallet.md [hunk 2]
```diff
@@ -221,24 +221,53 @@ For larger transactions, the wallet application provider will no longer automati
...
 
-If a user loses their mobile device but still has their cloud backup and signing device, they can use their PIN to unlock the cloud key. Then, with the help of the signing device, they can recover their wallet.
+Here are different scenarios where users may lose access to their wallet or keys and potential methods to resolve the problem:
 
-If they still have access to their mobile device but lost their purpose-built signing device, they can replace the lost key (rotate in a new key) with the help of the recovery key held by the auto-signer.
+#### The user loses their mobile device
```
Comment (danielnordh) [inline guide/savings-wallet.md:230]:
```suggestion
#### The user lost their mobile device
```

File: guide/savings-wallet.md [hunk 3]
```diff
@@ -221,24 +221,53 @@ For larger transactions, the wallet application provider will no longer automati
...
 
-However, if they lose both the mobile and signing device, they will only be able to recover their funds if they have also backed up either of the respective recovery phrases. [View prototype](https://www.figma.com/proto/hwispqET5MzsKUpn3i9iqD/Savings-account-case-study?page-id=1%3A55973&node-id=15%3A59302&viewport=-304%2C225%2C0.19710054993629456&scaling=scale-down&starting-point-node-id=1%3A56799&show-proto-sidebar=1).
+At some point, users are likely to either lose their mobile device, purchase a new one, or delete the wallet app.
+
+For them to recover their wallet again, they would need the following:
+1. Remember the sign-in details of the cloud account that contains the backup key
```
Comment (danielnordh) [inline guide/savings-wallet.md:235]:
```suggestion
1. The sign-in details of the cloud account that contains the backup key 
```

File: guide/savings-wallet.md [hunk 4]
```diff
@@ -221,24 +221,53 @@ For larger transactions, the wallet application provider will no longer automati
...
 
-If they still have access to their mobile device but lost their purpose-built signing device, they can replace the lost key (rotate in a new key) with the help of the recovery key held by the auto-signer.
+#### The user loses their mobile device
 
-However, if they lose both the mobile and signing device, they will only be able to recover their funds if they have also backed up either of the respective recovery phrases. [View prototype](https://www.figma.com/proto/hwispqET5MzsKUpn3i9iqD/Savings-account-case-study?page-id=1%3A55973&node-id=15%3A59302&viewport=-304%2C225%2C0.19710054993629456&scaling=scale-down&starting-point-node-id=1%3A56799&show-proto-sidebar=1).
+At some point, users are likely to either lose their mobile device, purchase a new one, or delete the wallet app.
+
+For them to recover their wallet again, they would need the following:
+1. Remember the sign-in details of the cloud account that contains the backup key 
+2. Remember the PIN they created on the wallet set up to decrypt the cloud key
```
Comment (danielnordh) [inline guide/savings-wallet.md:236]:
```suggestion
2. The PIN they created when setting up the wallet, to decrypt the cloud key
```

File: guide/savings-wallet.md [hunk 5]
```diff
@@ -221,24 +221,53 @@ For larger transactions, the wallet application provider will no longer automati
...
+3. Have access to their signing device
```
Comment (danielnordh) [inline guide/savings-wallet.md:237]:
```suggestion
3. Access to their signing device
```

File: guide/savings-wallet.md [hunk 6]
```diff
@@ -221,24 +221,53 @@ For larger transactions, the wallet application provider will no longer automati
...
+In these cases, they would need to have access to the following:
+1. The mobile device that set up the wallet
+2. The recovery key held by the auto-signer
+
+With these 2 things in place, they can simply rotate in a new key. However, if the user lost their mobile device as well, they would not be able to access their wallet again. [View prototype](https://www.figma.com/proto/hwispqET5MzsKUpn3i9iqD/Savings-account-case-study?page-id=1%3A55973&node-id=15%3A59302&viewport=-304%2C225%2C0.19710054993629456&scaling=scale-down&starting-point-node-id=1%3A56799&show-proto-sidebar=1).
 
 {% include image-gallery.html pages = page.images_recovery %}
 
-### Cloud storage key recovery & replacement
+#### The user loses access to their cloud account
```
Comment (danielnordh) [inline guide/savings-wallet.md:253]:
```suggestion
#### The user lost access to their cloud account
```

**Review by danielnordh — COMMENTED** [review]
LGTM

Some minor suggestion for brevity and clarity.

**Comment by steynviljoen67** [comment]
> LGTM
> 
> Some minor suggestion for brevity and clarity.

Thanks for the review and suggestions. I agree with all your suggestions.

**Comment by steynviljoen67** [comment]
@danielnordh not sure how it works but do you still need to merge this branch? Anything else I need to do?

**Pull request merged**